### PR TITLE
Mulitple UI fixes

### LINKED ISSUE #267: Top half of collection-edit page
The task entry area UI is not consistent with the rest of the page, need to also fix the log item that are on the collection page
## Detailed Description
The log items need to be resized so that there is not that much of white space. The trashcan button should only show up when the user hovers over the list item. The task entry area UI needs to use the same color way as the rest of the pages.
## Your Environment

* Browser Name and version:
* Operating System and version (desktop or mobile):

### PR DESCRIPTION
**Pull request by yuzilyu**
The top half of the collection-edit page is consistent with the rest of the UI of our webapp
I used new dark theme for the color, check it out and if you dont like it we can always revert it back
Added hover listener for the logitems
Logitems UI is refactored

### CODE REVIEW

**Comment by yuzilyu** [comment]
Let's actually not merge this pr for now until the Tuesday standup, I wanna demo the new Ui and gather opinions on the dark colorway

**Comment by thanhtong010** [comment]
Hi Yuzi, just a quick fix but on the Index page, could you add another period after the last R in RIVER (like R.I.V.E.R.)? I think that it's just a small typo since RIVER is an acronym

File: source/javascript/src/collection.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { CollectionItem, wrapper } from './components/CollectionItem.js'
 
-const addBtn = document.getElementById('add')
+const addBtn = document.getElementById('cb')
```
Comment (NoahT) [inline source/javascript/src/collection.js:3]: Yay the buttons are fixed ~(:

File: source/target/index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<html>
+    <link href="animation.css" rel="stylesheet">
+    <div class="loader loader--style2" style="color: #FF6700;" title="1">
```
Comment (NoahT) [inline source/target/index.html:3]: If the transition issue persists, we can add it as a task and put it on the backlog for this sprint.

**Review by NoahT — APPROVED** [review]
LGTM

File: source/target/index.html (hunk 2 above)
Comment (yuzilyu) [reply source/target/index.html:3]: I agree.

**Review by will-chung — APPROVED** [review]
Looks good! Excited to see the changes during our standup today 😄

**Review by fatIvan — COMMENTED** [review]
Looking good.

**Pull request merged**